### ISSUE TITLE
[Bug] Harvester not fully Harvesting

### ISSUE THREAD

**Issue by thatsIch** [powercrystals/MineFactoryReloaded#131]
Using MinefactoryReloaded 2.6.0.821

Using Harvester on Max Scale (Emeral Upgrade).

If I ever fly by my Wheat and Tree Farms, I get Seeds and Saplings. Since Harvester respect the leafs first and the rest is auto-collecting there shouldnt be any entities left.

With the current Setup I have, I get in avera 30 seeds and 10 saplings on every look I take at my farm.

**Comment by powercrystals** [comment]
Some saplings will always be left - some leaves decay before the harvester can get there.

As for seeds, I have no idea. It's probably another mod (natura/tconstruct) trying to boost seed output by spawning seeds on the ground. I can't really do anything about that. Closing since this is not actually a bug on our side.

**Comment by thatsIch** [comment]
Leaves decay since the trunk was removed until all leaves where removed. I am not sure how the harvester works completly but it seems to work in chunks, and often it happens to harvest it until a border and then left the rest of the leaves decay.

I dont have anything which boosts SeedOutput. NEither Natura/TConstruct or anything. The only thing it could be would been Forestry, but afaik it doesnt.

http://puu.sh/32T0d.png here is my total modlist, I have made this myself and gone through any config file and havent found a "seed" booster so far. If you know if one of these mods do, then I try to fix it myself.
